file.go

```diff
@@ -123,12 +145,24 @@ func (f *File) GetSection(name string) (*Section, error) {
 	}
 
 	sec := f.sections[name]
-	if sec == nil {
+	if sec == nil || len(sec) == 0 {
```

Comment:
`len` handles `nil`:
```suggestion
	if len(sec) == 0 {
```

Reply:
didn't know that, thank you